x/asset/ft/keeper/keeper_test.go

```diff
@@ -679,6 +679,330 @@ func TestKeeper_Burn(t *testing.T) {
 	requireT.ErrorIs(err, types.ErrDEXInsufficientSpendableBalance)
 }
 
+func TestKeeper_Burn_GovernanceDenom_Succeeds(t *testing.T) {
+	requireT := require.New(t)
+
+	testApp := simapp.New()
+	ctx := testApp.NewContextLegacy(false, tmproto.Header{})
+
+	ftKeeper := testApp.AssetFTKeeper
+	bankKeeper := testApp.BankKeeper
+	stakingKeeper := testApp.StakingKeeper
+
+	// Get bond denom
```

Comment:
remove the comments that are self-explanatory and can be easily 
understood from the code. there are few of such comments in tests.